### DISCUSSION TITLE
23.11.10 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#60] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 코드 리뷰대로 코드 수정
    - 테스트 코드 작성

**Comment by Jeonwoojae** [comment]
- 내일은 어떤 일을 하나요? 
  - 아티스트 저장 및 좋아요
  - 사용자와 코드 결합
